pkg/ansible/watches/watches.go

```diff
@@ -42,18 +38,18 @@ var log = logf.Log.WithName("watches")
 // Watch - holds data used to create a mapping of GVK to ansible playbook or role.
 // The mapping is used to compose an ansible operator.
 type Watch struct {
-	GroupVersionKind            schema.GroupVersionKind   `yaml:",inline"`
-	Blacklist                   []schema.GroupVersionKind `yaml:"blacklist"`
-	Playbook                    string                    `yaml:"playbook"`
-	Role                        string                    `yaml:"role"`
-	Vars                        map[string]interface{}    `yaml:"vars"`
-	MaxRunnerArtifacts          int                       `yaml:"maxRunnerArtifacts"`
-	ReconcilePeriod             time.Duration             `yaml:"reconcilePeriod"`
-	Finalizer                   *Finalizer                `yaml:"finalizer"`
-	ManageStatus                bool                      `yaml:"manageStatus"`
-	WatchDependentResources     bool                      `yaml:"watchDependentResources"`
-	WatchClusterScopedResources bool                      `yaml:"watchClusterScopedResources"`
-	Selector                    metav1.LabelSelector      `yaml:"selector"`
+	GroupVersionKind            schema.GroupVersionKind   `json:",inline"`
+	Blacklist                   []schema.GroupVersionKind `json:"blacklist"`
+	Playbook                    string                    `json:"playbook"`
+	Role                        string                    `json:"role"`
+	Vars                        map[string]interface{}    `json:"vars"`
+	MaxRunnerArtifacts          int                       `json:"maxRunnerArtifacts"`
+	ReconcilePeriod             metav1.Duration           `json:"reconcilePeriod"`
+	Finalizer                   *Finalizer                `json:"finalizer"`
+	ManageStatus                *bool                     `json:"manageStatus,omitempty"`
+	WatchDependentResources     bool                      `json:"watchDependentResources"`
```

Comment:
The Watch requires all values. The Alias struct which is used to do the parse needs the `omtempty` only. Also, it need to be changed for all attributes that are not mandatory in the watch files.